CHANGELOG.rst

```diff
@@ -47,6 +47,8 @@ LMS: Support adding students to a cohort via the instructor dashboard. TNL-163
 
 LMS: Show cohorts on the new instructor dashboard. TNL-161
 
+LMS: Extended hints feature
```

Comment:
So tiny for so much work :)
I wonder if there's any good documentation this could link to, to describe more thoroughly what this means.